core/iwasm/interpreter/wasm.h

```diff
@@ -484,6 +491,12 @@ typedef struct WASMTable {
 #endif
 } WASMTable;
 
+#if WASM_ENABLE_MEMORY64 != 0
+typedef uint64 linear_mem_ptr_t;
+#else
+typedef uint32 linear_mem_ptr_t;
```

Comment:
Not sure whether `linear_mem_offset_t` is better? Some code in loader and classic interpreter uses string `mem_offset` and `MEM_OFFSET`.

Reply:
I forgot to replace it before PR, now replace it with `mem_offset_t`